### ISSUE TITLE
 Walls both on level 19 attic and level 37 poolrooms can't hold blockstates( Blockdata) in the config

### ISSUE THREAD

**Issue by AnatomyKing** [PoiXson/pxnBackrooms-mirror#32]
I was implementing pxnBackrooms in my own server and I use note blocks with blockstates as you know. 

But for some reason when I use blockstates in WallA, WallB and in Wall on level 19 and level 37 It will crash my server. So I think it has trouble passing down the blockstates for these walls for some reason.

example config that will crash

```
Level_037:
  WallA: minecraft:note_block[instrument=hat,note=21,powered=false]
  WallB: minecraft:note_block[instrument=hat,note=19,powered=false]
```
  
```
  Level_019:
  Wall: minecraft:note_block[instrument=bass,note=4,powered=false]
```
 
 But it is quite odd since blockstates work in any other wall line of the config, for example when I use the blockstates in the basement wall or lobby wall they work flawless
 
```
 Level_001:
  Wall: minecraft:note_block[instrument=hat,note=18,powered=false]
```
  
```
  Level_000:
  Wall: minecraft:note_block[instrument=hat,note=9,powered=false]
  Wall-Base: minecraft:note_block[instrument=hat,note=15,powered=false]
```

### so it's specifically WallA and WallB from level 37 and Wall from level 19 that can't hold blockstates

here are the crash logs 

  WallA level 37
  https://mclo.gs/0ueEGB9
  
  WallB level 37
  https://mclo.gs/cmBgUPS

  Wall level 19
  https://mclo.gs/fmYlCCY

**Comment by AnatomyKing** [comment]
So my suggestion to fix the error, it look how the blockdata is passed down in the walls from  level 0 lobby and the level 1 basement and maybe you can see why those do work and these walls don't

**Comment by lorenzop** [comment]
fixed in 8a5b2d6
missed a few lines when updated to use block data, was still using Material types
